client/src/utils/savePacket.ts

```diff
@@ -0,0 +1,50 @@
+import { IMission, IDataPoint } from "./entities";
+import { IAprsTelemetryPacket } from "./TelemetryTypes";
+import api from "../services/api";
+
+export const saveAprsPacket = async (mission: IMission , packet: IAprsTelemetryPacket, packetNumber: number) => {
+    console.log('in save packet:', packet)
+    let missionId: string = '';
+
+    if (!mission._id) {
+        return 'No mission Id attached';
+    } else {
+        missionId = mission._id;
+    } 
```

Comment:
Checking for mission id here seems out of place. If `saveAprsPacket` is being called, shouldn't we know there is a mission w/ an id.

Reply:
the _id field is an optional paramter in all the entities that is only populated on a api call. this is just to make sure that an id is attach the the mission that is being passed in